spec/brcobranca/banco_mercantil_lote_spec.rb

```diff
@@ -0,0 +1,69 @@
+# -*- encoding: utf-8 -*-
+require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.